ppr-ui/src/views/discharge/DischargeRegistration.vue

```diff
@@ -185,60 +174,19 @@ export default defineComponent({
         goToDash()
         return
       }
+
+      // Conditionally load: could be coming back from confirm.
+      if (localState.fromConfirmation) {
+        return
+      }
```

Comment:
Copied this one from amendment Registration. It stops the data from being loaded again if returning from confirmation page.